test/utils/image/manifest.go

```diff
@@ -231,39 +231,39 @@ const (
 
 func initImageConfigs(list RegistryList) (map[ImageID]Config, map[ImageID]Config) {
 	configs := map[ImageID]Config{}
-	configs[Agnhost] = Config{list.PromoterE2eRegistry, "agnhost", "2.40"}
+	configs[Agnhost] = Config{list.PromoterE2eRegistry, "agnhost", "2.43"}
 	configs[AgnhostPrivate] = Config{list.PrivateRegistry, "agnhost", "2.6"}
 	configs[AuthenticatedAlpine] = Config{list.GcAuthenticatedRegistry, "alpine", "3.7"}
 	configs[AuthenticatedWindowsNanoServer] = Config{list.GcAuthenticatedRegistry, "windows-nanoserver", "v1"}
-	configs[APIServer] = Config{list.PromoterE2eRegistry, "sample-apiserver", "1.17.5"}
+	configs[APIServer] = Config{list.PromoterE2eRegistry, "sample-apiserver", "1.17.7"}
 	configs[AppArmorLoader] = Config{list.PromoterE2eRegistry, "apparmor-loader", "1.4"}
-	configs[BusyBox] = Config{list.PromoterE2eRegistry, "busybox", "1.29-2"}
+	configs[BusyBox] = Config{list.PromoterE2eRegistry, "busybox", "1.29-4"}
 	configs[CudaVectorAdd] = Config{list.PromoterE2eRegistry, "cuda-vector-add", "1.0"}
 	configs[CudaVectorAdd2] = Config{list.PromoterE2eRegistry, "cuda-vector-add", "2.2"}
 	configs[DistrolessIptables] = Config{list.BuildImageRegistry, "distroless-iptables", "v0.1.2"}
 	configs[Etcd] = Config{list.GcEtcdRegistry, "etcd", "3.5.5-0"}
```

Comment:
etcd should be `"3.5.5-1"`

Reply:
Does that need to get promoted still?

I don't see it when I run

**regctl tag ls registry.k8s.io/etcd**


/snip
3.4.9
3.4.9-0
3.4.9-1
3.5.0-0
3.5.0-beta.3-1
3.5.0-rc.0-0
3.5.1-0
3.5.3-0
3.5.4-0
3.5.5-0
3.6.0-alpha.0-0

Reply:
https://github.com/kubernetes/kubernetes/commit/ecd543be044f82b721ffaa1909f5a4fefe458a64#diff-de315dbb88d6d341b1682ef2fb0798723139fb4202ac607a334fc301e9359ec7R37

Reply:
https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-etcd/images.yaml
I can open a PR to promote 3.5.5-1

Reply:
> [kubernetes/k8s.io#4464](https://github.com/kubernetes/k8s.io/pull/4464)

Approve/LGTM-ed

Reply:
updated!